src/commands/install-sfdx.yml

```diff
@@ -8,32 +8,64 @@ description: >
   'Install the Salesforce "sfdx" cli utility giving access to the "sfdx" command.'
 parameters:
   version:
-    description: 'By default, the latest version of the standalone CLI will be installed. To install via npm, supply a version tag such as "latest" or "6".'
+    description: 'By default, the latest version of the standalone CLI will be installed on linux. To install via npm, supply a version tag such as "latest" or "6".  For windows, only npm installation is supported and omitting the version will give you "latest"'
     default: ''
     type: string
+  os:
+    description: operating system to run tests on
+    type: enum
+    enum: ['linux', 'windows']
+    default: 'linux'
 
+# there are four combos of OS and version
 steps:
-  # when version is defined.
   - when:
-      condition: << parameters.version >>
+      condition:
```

Comment:
Personally, I don't think there's any reason to have a conditional here. I think it's far simpler to install via npm regardless of the os

Reply:
Aside from OS, is there any reason that we would want to run tests against an artifact vs result of npm install?

Reply:
it's nice to be able to run from the artifact to test it.  W-8721307:
> Runs the tests on an npm installed cli and artifact installed cli

Reply:
shouldn't the OS of the machine/executor determine what OS to install it for? If we pass it in as an arg we have to create a lot more logic when calling it and making sure that we call it with the correct OS. I think it'd make more sense to have args for install type (npm v archive) and then channel (stable, latest, latest-rc, stable-rc)